rfc/src/rfcs/0010-quantifiers.md

```diff
@@ -0,0 +1,207 @@
+- **Feature Name:** Quantifiers
+- **Feature Request Issue:** [#2546](https://github.com/model-checking/kani/issues/2546) and [#836](https://github.com/model-checking/kani/issues/836)
+- **RFC PR:** [#](https://github.com/model-checking/kani/pull/)
+- **Status:** Unstable
+- **Version:** 1
+
+-------------------
+
+## Summary
+
+Quantifiers are like logic-level loops and a powerful reasoning helper, which allow us to express statements about objects in a domain that satisfy a given condition. There are two primary quantifiers: the existential quantifier (∃) and the universal quantifier (∀).
```

Comment:
That first part of the sentence trips me up: this is perhaps too simplistic a language?

Reply:
I removed the first sentence.